src/allencell_ml_segmenter/training/view.py

```diff
@@ -296,18 +297,46 @@ def getTypeOfWork(self) -> str:
         return "Training"
 
     def showResults(self) -> None:
```

Comment:
I see no references to `showResults`.  How does it get called?

Reply:
see `/core/view.py`: https://github.com/AllenCell/allencell-segmenter-ml/blob/9756bd3565fedf27702cedfc1af4151da196e6d3/src/allencell_ml_segmenter/core/view.py#L90-L94